lib/loadQuestions.js

```diff
@@ -1,20 +1,13 @@
-const request = require('request');
+const axios = require('axios');
 const parse = require('./parse');
 
-const loadQuestions = formId => {
-  return new Promise((resolve, reject) => {
-    request(
-      `https://docs.google.com/forms/d/e/${formId}/viewform`,
-      (error, response, body) => {
-        if (error) {
-          reject(error);
-        } else {
-          let questions = parse(body);
-          resolve(questions);
-        }
-      }
-    );
+const loadQuestions = async formId => {
+  const body = await axios({
+    url: `https://docs.google.com/forms/d/e/${formId}/viewform`,
   });
-};
+
+  const questions = await parse(body.data);
```

Comment:
use `res` instead of `body`:

```suggestion
  const questions = await parse(res.data);
```